otter_buddy/cogs/interview_match.py

```diff
@@ -178,6 +199,36 @@ async def stop(self, ctx):
         else:
             msg = "No activity was running! 😱"
         await ctx.send(msg)
+    
+    @interview_match.group(  # type: ignore
+    brief="Commands related to trigger manually the Interview Match activity!",
+    invoke_without_command=True,
+    )   
+    @commands.has_any_role(OTTER_ADMIN, OTTER_MODERATOR)
+    async def run(self, ctx) -> None:
+        """
+        Admin commands related to trigger and test the activity
+        """
+        await ctx.send_help(ctx.command)
+
+    @run.command(brief="Admin command to trigger the send weekly message job")  # type: ignore
```

Comment:
What the difference between this command and the one below, I can see the difference but looks like the documentation it's the same.